lib/entry.js

```diff
@@ -5,6 +5,8 @@ const { messages } = elasticio;
 const { callJSForceMethod } = require('./helpers/wrapper');
 const { getLinkedObjectTypes, processMeta } = require('./helpers/utils');
 
+const DEFAULT_MAX_FETCH = 10000;
```

Comment:
I'd suggest renaming the constant to something self-documenting. Like MAX_FETCH_ALLOWED_BY_SALESFORCE